src/pyosmeta/github_api.py

```diff
@@ -133,8 +133,6 @@ def handle_rate_limit(self, response):
         If the remaining requests are exhausted, it calculates the time
         until the rate limit resets and sleeps accordingly.
         """
-        print("hello there")
```

Comment:
this might have been from fixing tests 😆 it looks like something i would add (and then forget about).